src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/JavaThread.java

```diff
@@ -481,6 +481,25 @@ public Address lastSPDbg() {
     return access.getLastSP(addr);
   }
 
+  // Print the contents of all registers in the thread's context
+  public void printThreadContextOn(PrintStream out, boolean verbose){
+    ThreadContext tc = getThreadProxy().getContext();
+    for (int r = 0; r < tc.getNumRegisters(); r++) {
+      Address regAddr = tc.getRegisterAsAddress(r);
+      System.out.format("Register(%s): %s", tc.getRegisterName(r), regAddr);
```

Comment:
Would you consider just printing name = value, rather than printing the word Register many times? 8-)

Reply:
Ok. I changed it to no longer print "Register"